src/client/components/UserPage/Drawer/ControlPanel/widgets/DownloadButton.tsx

```diff
@@ -6,12 +6,49 @@ import {
   MenuItem,
   Typography,
 } from '@material-ui/core'
+import FileSaver from 'file-saver'
 
 import TrailingButton from './TrailingButton'
 import downloadIcon from '../assets/download-icon.svg'
-import { useDrawerState, useDrawerDispatch } from '../..'
-import QRCodeModal from '../QRCodeModal'
-import DrawerActions from '../helpers/reducers'
+import { useDrawerState } from '../..'
+import ImageFormat from '../../../../../../shared/util/imageFormat'
```

Comment:
```suggestion
import ImageFormat from '../../../../../../shared/util/image-format'
```
nitpick filename, should not be camelCase

Reply:
Will make the change!